### PR TITLE
Modify rule S5704: LaYC format

### PR DESCRIPTION
**Pull request by maksim-grebeniuk-sonarsource**
## Review

A dedicated reviewer checked the rule description successfully for:

- [ ] logical errors and incorrect information
- [ ] information gaps and missing content
- [ ] text style and tone
- [ ] PR summary and labels follow [the guidelines](https://github.com/SonarSource/rspec/#to-modify-an-existing-rule)

### CODE REVIEW

File: rules/S5704/python/rule.adoc [hunk 1]
```diff
@@ -1,16 +1,18 @@
+This rule raises an issue when a bare ``++raise++`` statements is in a ``++finally++`` block.
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S5704/python/rule.adoc:1]: statements -> statement

File: rules/S5704/python/rule.adoc [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+
 == Why is this an issue?
 
 A bare ``++raise++`` statement, i.e. a ``++raise++`` with no exception provided, will re-raise the last active exception in the current scope. If no exception is active a ``++RuntimeError++`` is raised instead.
 
 If the bare "raise" statement is in a ``++finally++`` block, it will only have an active exception to re-raise when an exception from the ``++try++`` block is not caught or when an exception is raised by an ``++except++`` or ``++else++`` block. Thus bare ``++raise++`` statements should not be relied upon in ``++finally++`` blocks. It is simpler to let the exception raise automatically.
```
Comment (guillaume-dequenne-sonarsource) [inline rules/S5704/python/rule.adoc:7]: I vaguely have the feeling that we could be a bit more explicit, saying that in other cases, no exception will be active, leading to the `RuntimeError`.
I also have the feeling we could say explicitly that without any `raise` statement in the block, the exception will propagate automatically anyway (as it is implied in the last sentence, but not very explicitly).

And at the same time I think that the current description is very clear and straight to the point, and would want to make it less readable.

Just sharing my thoughts without asking for changes. Wdyt @jean-jimbo-sonarsource?

**Review by guillaume-dequenne-sonarsource — APPROVED** [review]
Looking good!
A minor typo to fix, and a more general comment (which I'm not asking to address, but if you do, let's have a 2nd round of review).

File: rules/S5704/python/rule.adoc (hunk 2 above)
Comment (jean-jimbo-sonarsource) [reply rules/S5704/python/rule.adoc:7]: I really like how concise it is .

**Review by jean-jimbo-sonarsource — APPROVED** [review]

**Pull request merged**